Foxtrot/Tests/Properties/AssemblyInfo.cs

```diff
@@ -45,3 +46,5 @@
 // by using the '*' as shown below:
 [assembly: AssemblyVersion("1.0.0.0")]
 [assembly: AssemblyFileVersion("1.0.0.0")]
+
+[assembly: CollectionBehavior(CollectionBehavior.CollectionPerAssembly, DisableTestParallelization = true)]
```

Comment:
The tests manipulate static state. MSTest was never parallelized so this attribute is just included to match that behavior. I could add a comment.